api/envoy/data/accesslog/v2/accesslog.proto

```diff
@@ -54,6 +56,15 @@ message HTTPAccessLogEntry {
   HTTPResponseProperties response = 4;
 }
 
+// Defines fields for a connection
+message ConnectionProperties {
+  // Number of bytes received from downstream.
+  uint64 bytes_received = 1;
```

Comment:
Can we switch the naming of these fields to `<foo>_bytes`? This would make it consistent with other bytes quantity fields in this file.